shoppingm8_fe/lib/user/accountManagementWidget.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:dio/dio.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:shoppingm8_fe/auth/authenticationApiProvider.dart';
+import 'package:shoppingm8_fe/auth/loginWidget.dart';
+import 'package:shoppingm8_fe/user/userDto.dart';
+
+class AccountManagementWidget extends StatefulWidget {
+  final String serverUrl;
+  final Dio dio;
+  
+  AccountManagementWidget({this.serverUrl, this.dio});
+
+  @override
+  _AccountManagementWidgetState createState() => _AccountManagementWidgetState(dio: dio, serverUrl: serverUrl);
+}
+
+class _AccountManagementWidgetState extends State<StatefulWidget> {
+  final Dio dio;
+  final String serverUrl;
+
+  UserDto _userDto;
+
+  AuthenticationApiProvider _apiProvider;
+  _AccountManagementWidgetState({this.dio, this.serverUrl}) {
+    _apiProvider = AuthenticationApiProvider(serverUrl);
+    _showUser();
+  }
+
+  Future<UserDto> _showUser() async {
+    Response response = await _apiProvider.me(dio);
+    if (response.statusCode == 200) {
+      setState(() {
+        _userDto = UserDto.fromJson(response.data);
+      });
+    }
+  }
+
+  ImageProvider _displayUserImage(String imageUrl) {
+    if (imageUrl != null)
+      return NetworkImage(imageUrl);
```

Comment:
Please fix the url by concatenation with serverUrl

Reply:
As above.

Reply:
Changed